src/Data/Project.hs

```diff
@@ -56,5 +57,5 @@ readProjectFromPaths maybeRoot path lang excludeDirs = do
   blobs <- liftIO $ traverse (readBlobFromFile' . toFile) paths
   pure $ Project (Path.toString rootDir) blobs lang (fmap Path.toString excludeDirs)
```

Comment:
`Project` also needs a dose of `pathtype`, but I’ve chosen to do that in a separate PR, both because this patch is big enough already and because I think an OSS person wants to help out with that issue, which is fine by me, as I’ve written this patch like five times and I’m tired of it. 😇